arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -177,34 +178,38 @@ arangodb::aql::AqlValue aqlFnTokens(arangodb::aql::ExpressionContext* expression
   }
 
   if (!pool) {
+    auto n = std::string(name.c_str(), name.size());
     auto const message = "failure to find arangosearch analyzer with name '"s +
-                         name + "'" + while_tokens;
+                         n + "'" + while_tokens;
```

Comment:
how about `std::string("failure to find ...").append(name.c_str(), name.size()).append("'")` etc